docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md

```diff
@@ -0,0 +1,63 @@
+# get_raw_transaction
+The  `get_raw_transaction`  method returns hexadecimal encoded signed transactions.
+
+## Arguments
+|Structure|Type|Description|
+|:-------:|:---:|:--------:|
+|coin|string|the name of the coin the user desires to request for the transaction |
+|tx_hash|string|hash of the transaction|
+
+## Response
+|Structure|Type|Description|
+|:--------:|:----:|:------:|
+|tx_hex|BytesJson|raw bytes of signed transaction|
+
+
+## Examples:
+
+### Request
```

Comment:
Wrap responses with 

```

<div style="margin-top: 0.5rem;">
<collapse-text hidden title="Response">
```
Responses go here

```
</collapse-text>
</div>```

This adds a button to show/hide he responses like
![image](https://user-images.githubusercontent.com/35845239/162945548-eb129952-5705-4d1a-83aa-8ae2694980cf.png)